models/br_ms_sinasc/code/br_ms_sinasc_etl.py

```
@@ -0,0 +1,694 @@
+    Path
+        Caminho local do arquivo baixado.
+    """
+    url = URL_TEMPLATE.format(uf=uf, ano=ano)
+    destino = INPUT_DIR / f"DN{uf}{ano}.dbc"
+    if destino.exists():
+        logger.info("Arquivo já existe, download ignorado: %s", destino.name)
+    else:
+        logger.info("Baixando %s", url)
+        urllib.request.urlretrieve(
+            url, filename=str(destino), reporthook=_reporthook
+        )
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)